### PR TITLE
L1T offine DQM changes for pp reference run

### PR DESCRIPTION
**Pull request by thomreis**
To make the L1T offline DQM work for the pp reference run the trigger paths used must be adapted for the pp reference run HLT menu.

This PR includes commit https://github.com/cms-sw/cmssw/pull/21122/commits/6be5e63e80d043519f48e937d92d5d61e2f87bae from PR #21122 to get the era "Run2_2017_ppRef".

### CODE REVIEW

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
please test

**Comment by rekovic** [comment]
+1

**Comment by dmitrijus** [comment]
+1

**Comment by thomreis** [comment]
@davidlange6 can this be merged please? We would like to have it for the pp reference run.

File: DQMOffline/L1Trigger/python/L1TMuonDQMOffline_cff.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+from DQMOffline.L1Trigger.L1TMuonDQMOffline_cfi import *
+
+# modifications for the pp reference run
+muonEfficiencyThresholds_HI = [5, 7, 12]
```
Comment (davidlange6) [inline DQMOffline/L1Trigger/python/L1TMuonDQMOffline_cff.py]: hi @thomreis - please move these lines into the file that defines l1tMuonDQMOffline

Comment (davidlange6) [reply DQMOffline/L1Trigger/python/L1TMuonDQMOffline_cff.py]: [from line 5 to 14]

Comment (thomreis) [reply DQMOffline/L1Trigger/python/L1TMuonDQMOffline_cff.py]: I had that before and thought it should not be in a cfi. I will move it back.

**Comment by davidlange6** [comment]
Thanks -module-level era modifications should be in the same file as the module they modify

**Comment by thomreis** [comment]
@davidlange6 done. I will take care of the backports now.

**Comment by rekovic** [comment]
please test

**Comment by davidlange6** [comment]
merge

**Pull request merged**